sampleapp/Cargo.toml

```diff
@@ -8,4 +8,4 @@ edition = "2018"
 
 [dependencies]
 glium = "0.26"
-obj-rs = { path = "../obj-rs", default-features = false, features = ["glium"] }
+obj-rs = { path = "../obj-rs", default-features = false, vulkano=false, features = ["glium"] }
```

Comment:
I get this warning from this change. Was there any reason to add this key?

```
warning: /<path>/obj-rs/sampleapp/Cargo.toml: unused manifest key: dependencies.obj-rs.vulkano
```

Reply:
Nope! that's just something I forgot to remove when debugging! I'll remove the key and update the PR.